### PR TITLE
Use system default flag emoji by default

### PR DESCRIPTION
**Pull request by andychow326**
ref DEV-2000

image

### CODE REVIEW

**Review by louischan-oursky — CHANGES_REQUESTED** [review]
@andychow326 Please fix the build error.

**Comment by andychow326** [comment]
> @andychow326 Please fix the build error.

Fixed, thanks! :adore2x:

**Review by louischan-oursky — CHANGES_REQUESTED** [review]

**Review by louischan-oursky — CHANGES_REQUESTED** [review]
```
diff --git a/authui/src/authflowv2/components/phone-input.css b/authui/src/authflowv2/components/phone-input.css
index c6d218f3f..66c4e130b 100644
--- a/authui/src/authflowv2/components/phone-input.css
+++ b/authui/src/authflowv2/components/phone-input.css
@@ -5,7 +5,7 @@
     --phone-input__line-height: var(--typography-body-large__line-height);
     --phone-input__letter-spacing: var(--typography-body-large__letter-spacing);
     --phone-input__font-weight: var(--typography-body-large__font-weight);
-    --phone-input__font-family: twemoji-countries, inter;
+    --phone-input__font-family: var(--typography-body-large__font-family);
     --phone-input__space-x: 0.5em;
 
     --phone-input__trigger-ring-color: var(--color-neutral-100);
```

Please add this change as well.

File: resources/authgear/templates/en/web/authflowv2/__html_head.html [hunk 1]
```diff
@@ -59,8 +59,6 @@
 
 <link rel="preconnect" href="https://fonts.googleapis.com">
 <link rel="preload" href='{{ call $.GeneratedStaticAssetURL "material-symbols-outlined-subset.woff2" }}' as="font" crossorigin="anonymous">
-<link rel="preload" href='{{ call $.GeneratedStaticAssetURL "Twemoji.Mozilla-subset.woff2" }}' as="font" crossorigin="anonymous">
```
[the remark this answers is not in the document]
Comment (louischan-oursky) [reply resources/authgear/templates/en/web/authflowv2/__html_head.html:62]: Per our offline discussion, we should not preload the font here. Instead, our vite plugin should be smart enough to output the preload elements to load our **actually used fonts**.

**Comment by andychow326** [comment]
@louischan-oursky Done!

**Review by louischan-oursky — APPROVED** [review]

**Pull request merged**